src/gengowatcher/web.py

```
@@ -0,0 +1,893 @@
+        sections = []
+        # The config.config is a dict with section keys
+        for section_name, section_data in self.config.config.items():
+            if isinstance(section_data, dict):
+                sections.append(ConfigSection(section=section_name, options=section_data))
+        return sections
+
+    def update_config(self, section: str, option: str, value: str) -> bool:
+        """Update configuration value."""
+        try:
+            self.watcher.set_config_value(section, option, value)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)